docs/user/types/references/reference/summary-usage.en.md

```diff
@@ -9,4 +9,4 @@ For different versions of Advisories it has the following format:
 
   `ID: - TITLE - FORMAT VERSION`
 
-where `ID` is the value of `document/tracking/id`, `TITLE` is optional and the value of `document/title`, `FORMAT` is one of `PDF`, `TXT`, `CSAF` or `HTML` and `VERSION` is the document's version.
+where `ID` is the value of `/document/tracking/id`, `TITLE` is optional and the value of `document/title`, `FORMAT` is one of `PDF`, `TXT`, `CSAF` or `HTML` and `VERSION` is the document's version.
```

Comment:
The version should always be the latest - therefore, usually there is no need to announce that.
```suggestion
`ID: TITLE - FORMAT `

where `ID` is the value of `/document/tracking/id`, `TITLE` is optional and the value of `/document/title` and `FORMAT` is one of `PDF`, `TXT`, `CSAF` or `HTML`.

If different versions of the document reside under different URLs, the value of `/document/tracking/version` should be appended as ` - version VERSION` after `FORMAT`. If the latest version is always at the same URL is such case, the `VERSION` part can be identified as `- latest version`

Reply:
updated